format/FormatHDF5Exceptions.py

```diff
@@ -0,0 +1,34 @@
+import h5py
+
+# static enumerations for useless file types
+
+HDF5_UNKNOWN = -1
+HDF5_NXS_FILE = 0
+HDF5_DATA_FILE = 1
+HDF5_META_FILE = 2
+HDF5_MPCCD = 0
```

Comment:
I wonder if encapsulation using the [enum](https://docs.python.org/3/library/enum.html) library might be preferable?

Reply:
👍 

TIL this